### PR TITLE
Added kube token instructions

### PR DESCRIPTION
**Pull request by sjletcher**

### CODE REVIEW

File: how-to-docs/kubernetes-token.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# How to get a kubernetes token for the UK cluster
+
```
Comment (Joseph-Irving) [inline how-to-docs/kubernetes-token.md]: capitalise Kubernetes, it's a proper noun

File: how-to-docs/kubernetes-token.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Log into the [platform hub][platform hub link].
+
+Select **Support Requests** under **Help & Support** from the sidebar.
+
+Select _Kubernetes token request_ from the list of available request templates.
+
+Select the required cluster from the drop down list and submit your request.
```
Comment (tasharnvb) [inline how-to-docs/kubernetes-token.md]: shouldn't cluster be namespace?

File: how-to-docs/kubernetes-token.md (hunk 1 above)
Comment (sjletcher) [reply how-to-docs/kubernetes-token.md]: Thank you

File: how-to-docs/kubernetes-token.md (hunk 2 above)
Comment (sjletcher) [reply how-to-docs/kubernetes-token.md]: Yes, thank you

Comment (jits) [reply how-to-docs/kubernetes-token.md]: The existing support request form is outdated (so it needs updating); the only thing a user needs to provide now is what cluster they would like access to. Actual role bindings to namespaces is done separately (as a PR to the relevant Gitlab repo).

In the future hopefully we'll combine these two steps into one flow in the hub. But for now they are separate.

**Comment by oyelekci** [comment]
Thanks, looks good. As per my comments on Tasharn's readme, I think we also need a separate doc for devops team members along the lines of https://github.com/UKHomeOffice/application-container-platform/blob/master/how-to-docs/artifact-robot.md or https://github.com/UKHomeOffice/application-container-platform/blob/master/how-to-docs/onboarding.md but for the UK cluster

**Pull request merged**